### PR TITLE
Link to registration on home page if unregistered

### PR DESCRIPTION
**Pull request by tcluri**

### CODE REVIEW

**Comment by punchagan** [comment]
Thanks for the PR!

- Can you post screenshots?
- Can you fix the rebase? The createEffect stuff duplicates the `onMount` stuff that's already there in `main`.

**Comment by tcluri** [comment]
post_registration_link
pre_registration

File: frontend/src/components/Home.js [hunk 1]
```diff
@@ -34,11 +34,48 @@ const Home = () => {
     }
   };
 
+
+  fetch("/api/user", {
```
Comment (punchagan) [inline frontend/src/components/Home.js:38]: Why is this function still here? It's been moved to a util and is called on mount?

Comment (tcluri) [reply frontend/src/components/Home.js:38]: Oh, okay

File: frontend/src/components/Home.js [hunk 2]
```diff
@@ -39,6 +39,23 @@ const Home = () => {
...
+        <div>
+          <Show when={!store.data.player}>
+             <h3 class="text-2xl font-bold mb-4">
+              Want to play?{" "}
+              <A href="/registration" class="font-medium text-blue-600 dark:text-blue-500 hover:underline">
+                Register!
+              </A>
+            </h3>
+          </Show>
+          <Show when={store.data.player}>
```
Comment (punchagan) [inline frontend/src/components/Home.js:52]: This can become a `fallback` to the Show above?

Comment (tcluri) [reply frontend/src/components/Home.js]: Added a fallback to the `<Show when={!store.data.player}...>`

**Pull request merged**